flair/data.py

```diff
@@ -175,7 +175,7 @@ def __init__(self, text: str = None, use_tokenizer: bool = False, labels: List[s
                 self.add_token(Token(word))
 
     def __getitem__(self, token_id: int) -> Token:
-        return self.get_token(token_id)
+        return self.get_token(token_id + 1)
```

Comment:
get_token does a full iteration over all tokens in the sentence to identify the one with the right id. 

perhaps more efficient would be: 

`token = self.tokens[idx]`

that saves us the iteration.

Reply:
to clarify, the whole __getitem__ would then be: 

```python
    def __getitem__(self, idx: int) -> Token:
        token = self.tokens[idx]
        if token is None:
            raise IndexError()
        return token
```